### PR TITLE
virus_plowing

### PR DESCRIPTION
**Pull request by duck123acb**
# Your checklist for this pull request

## About your game

What is your game about? 
> Plowing infected spots on the board with a shield.

How to play your game?
> Press WASD to move, Push the shield (blue) into the infected tiles (green) to plow them away. Hit ‘I’ for Info.

## Code
Check off the items that are true.
- [x] The game was made using the [Sprig editor](https://editor.sprig.hackclub.com/).
- [x] The game is placed in the in the [`/games` directory](https://github.com/hackclub/sprig/tree/main/games).
- [x] The code is significantly different from all other games in the [Sprig gallery](https://sprig.hackclub.com/gallery) (except for games labeled "demo").
- [x] The game runs without errors. 
- [x] The name of the file/game contains only alphanumeric characters, `-`s, or `_`s.
- [x]The game name is not the same as the others from [gallery](https://sprig.hackclub.com/gallery)

## Image (If an image is used)

- [x] The image is in the [`/games/img` directory](https://github.com/hackclub/sprig/tree/main/games/img).
- [x] The name of the image matches the name of your file. Example: `sokoban_plus.js` -> `sokoban_plus.png`.

> Thanks for PR!

### CODE REVIEW

File: games/virus_plowing.js [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+..................
+..................
+..................
+..................
+..................
+..................
+..................
+..................
+..................
+..................`,
```
Comment (LucasHT22) [inline games/virus_plowing.js:152]: what is the function of this empty map?

Comment (duck123acb) [reply games/virus_plowing.js:152]: The function of the empty map is for the game over screen

Comment (LucasHT22) [reply games/virus_plowing.js:152]: oh, right!

**Comment by LucasHT22** [comment]
I'll do just one more test and I'll merge

**Review by LucasHT22 — APPROVED** [review]
Looks good to me! Congrats:)
If you're a teen, you can get your sprig here! https://sprig-order.hackclub.dev/

out of curiosity, how did you meet sprig?

**Comment by duck123acb** [comment]
I learned about sprig while reading an online article

It can be found here: [https://www.geeky-gadgets.com/open-source-game-console-11-11-2022/](https://flip.it/PJCsmv)

**Comment by duck123acb** [comment]
Thank you for your time, have a wonderful day!

**Comment by LucasHT22** [comment]
thanks for contributing to the Sprig!

**Pull request merged**